### PR TITLE
review feedback and updates for 2.4 release notes

### PR DESCRIPTION
**Pull request by lmandavi**
This PR incorporates feedback received from @dankenigsberg, @nellyc , @orenc1 , and @aburdenthehand for the 2.4 release notes snapshot that was circulated for review.

A snapshot of the 2.4 Release Notes was circulated for review to all the leads on Friday with a PR associated with it. That PR was merged this morning by @aburdenthehand  in order to resolve merge conflicts. In response to the feedback received in that PR, this PR was generated to capture that feedback and incorporate it.

Verified with @aburdenthehand - this PR goes straight to Peer Review now.

### CODE REVIEW

File: virt/virt-2-4-release-notes.adoc [hunk 1]
```diff
@@ -102,7 +102,7 @@ from being inadvertently started before an upload has completed.
 //Placeholder for new content.
 
 //CNV-5465-Disconnected installation
-* {VirtProductName} can be installed on disconnected clusters in restricted network environments that do not have Internet connectivity. You can create local mirrors for the {VirtProductName} Operator, and install the Operator from a local catalog image that is accessible to the disconnected cluster. Learn more about xref:../operators/olm-restricted-networks.adoc#olm-understanding-operator-catalog-images_olm-restricted-networks[using Operator Lifecycle Manager on restricted networks].
+* {VirtProductName} can be installed on disconnected clusters in restricted network environments that do not have Internet connectivity. You can create local mirrors for the {VirtProductName} Operator, and install the Operator from a local catalog image that is accessible to the disconnected cluster. Learn more about xref:https://access.redhat.com/articles/4827691[Installing OpenShift Virtualization on a restricted network cluster].
```
Comment (aburdenthehand) [inline virt/virt-2-4-release-notes.adoc:105]: We're linking to external content so this needs to be a 'link'.
ie:  Learn more about link:https://access.redhat.com/articles/4827691[Installing OpenShift Virtualization on a restricted network cluster].

Comment (kalexand-rh) [reply virt/virt-2-4-release-notes.adoc:105]: +1, this should fix that build error.

File: virt/virt-2-4-release-notes.adoc [hunk 2]
```diff
@@ -182,19 +182,9 @@ value by running `oc describe node <node>` for all nodes and looking at the
...
-
 //Jira story for the following content https://issues.redhat.com/browse/CNV-4757. Nelli noted that for CNV 2.5, this can be removed from Known Issues. The defect status, however, is CLOSED.
 * {VirtProductName} cannot reliably identify node drains that are triggered by
 running either `oc adm drain` or `kubectl drain`. Do not run these commands on
 the nodes of any clusters where {VirtProductName} is deployed. The nodes might not
 drain if there are virtual machines running on top of them.
-The current solution is to put nodes into maintenance.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1707427[*BZ#1707427*])
-
+The current solution is to xref:../virt/node_maintenance/virt-setting-node-maintenance.adoc[put nodes into maintenance].
```
Comment (kalexand-rh) [inline virt/virt-2-4-release-notes.adoc:190]: Looks like you're off one level here and missing the anchor. Try `xref:../../virt/node_maintenance/virt-setting-node-maintenance.adoc#virt-setting-node-maintenance`

Comment (lmandavi) [reply virt/virt-2-4-release-notes.adoc]: Added the anchor, thanks.

File: virt/virt-2-4-release-notes.adoc (hunk 1 above)
Comment (lmandavi) [reply virt/virt-2-4-release-notes.adoc]: Done.

File: virt/virt-2-4-release-notes.adoc [hunk 3]
```diff
@@ -102,7 +102,7 @@ from being inadvertently started before an upload has completed.
 //Placeholder for new content.
 
 //CNV-5465-Disconnected installation
-* {VirtProductName} can be installed on disconnected clusters in restricted network environments that do not have Internet connectivity. You can create local mirrors for the {VirtProductName} Operator, and install the Operator from a local catalog image that is accessible to the disconnected cluster. Learn more about xref:../operators/olm-restricted-networks.adoc#olm-understanding-operator-catalog-images_olm-restricted-networks[using Operator Lifecycle Manager on restricted networks].
+* {VirtProductName} can be installed on disconnected clusters in restricted network environments that do not have Internet connectivity. You can create local mirrors for the {VirtProductName} Operator, and install the Operator from a local catalog image that is accessible to the disconnected cluster. Learn more about https://access.redhat.com/articles/4827691[Installing OpenShift Virtualization on a restricted network cluster].
```
Comment (kalexand-rh) [inline virt/virt-2-4-release-notes.adoc:105]: s/Operator,/Operator

**Review by kalexand-rh — COMMENTED** [review]
One typo, then please squash.

**Comment by lmandavi** [comment]
@kalexand-rh - squashed the commits. Adding @aburdenthehand for awareness. Thanks!

**Pull request merged**